cmd/tk/workflow.go

```diff
@@ -137,6 +137,16 @@ func showCmd() *cobra.Command {
 	}
 	vars := workflowFlags(cmd.Flags())
 	cmd.Run = func(cmd *cobra.Command, args []string) {
+		fi, err := os.Stdout.Stat()
+		if err != nil {
+			log.Fatalln("stdout pipe detection:", err)
+		}
+
+		if fi.Mode()&os.ModeCharDevice == 0 {
+			fmt.Fprintf(os.Stderr, "⚠️  Do not pipe the output of tk show. Use tk apply instead ⚠️\n")
```

Comment:
We do not use emojis in the messages so far and I would like to stick with ASCII only.

Reply:
Sorry, I just copied it from the original ticket :slightly_smiling_face: I changed the message.